identity-service/src/routes/healthCheck.js

```diff
@@ -237,14 +237,16 @@ module.exports = function (app) {
     let funderBalance = parseFloat(Web3.utils.fromWei(await getEthRelayerFunds(funderAddress), 'ether'))
     let funderAboveMinimum = funderBalance >= minimumFunderBalance
     let belowMinimumBalances = []
-    let balances = []
-    for (let account of ETH_RELAY_HEALTH_ACCOUNTS) {
-      let balance = parseFloat(Web3.utils.fromWei(await getEthRelayerFunds(account), 'ether'))
-      balances.push({ account, balance })
-      if (balance < minimumBalance) {
-        belowMinimumBalances.push({ account, balance })
-      }
-    }
+
+    const balances = await Promise.all(
+      [...ETH_RELAY_HEALTH_ACCOUNTS].map(async account => {
```

Comment:
you can't just do `ETH_RELAY_HEALTH_ACCOUNTS.map()`?

Reply:
it's a set

Reply:
not sure if intended behavior to dedupe what's in the args, but it's a subtle (maybe important behavior)

Reply:
ah i didnt see it was a set, makes sense
also yeah de-duping seems like the right move here as well

thanks for clarifying